emain/emain-window.ts

```diff
@@ -699,9 +699,25 @@ ipcMain.on("delete-workspace", (event, workspaceId) => {
             console.log("user cancelled workspace delete", workspaceId, ww?.waveWindowId);
             return;
         }
+        
+        if (workspaceList.length > 1 && ww?.workspaceId == workspaceId){
```

Comment:
This should be combined with the if statement on line 720. The behavior should be:

```typescript
if (ww?.workspaceId == workspaceId) {
    if (workspaceList.length > 1){
        // switch workspace
        ...
    } else {
        // delete workspace
        ...
    }
}
```

Reply:
Also, while it's extremely unlikely, workspaceList returned by `WorkspaceService.ListWorkspaces` could be null so we should probably have `workspaceList?.length`